docs/Researcher/cli-reference/runai-submit.md

```diff
@@ -357,8 +370,17 @@ runai submit --job-name-prefix -i gcr.io/run-ai-demo/quickstart -g 1
 
 #### -s | --service-type `<string>`
 
-> External access type to interactive jobs. Options are: portforward (deprecated), loadbalancer, nodeport, ingress.
-  
+> External access type to interactive jobs. Options are:
+    > * `portforward` (deprecated)
```

Comment:
If it is deprecated, don't we want to omit it?

Reply:
Usually when something is deprecated, it means you can still use it, but there is a better option which works better. Not always does it mean it does not work. Sometimes there are scripts that use it so we need to let the users know that it is being phased out. Once it is removed we then take it out of the docs and depending upon the effect it has on the users declare it in the release notes.